_config.yml

```diff
@@ -104,4 +105,4 @@ defaults:
     scope:
       path: ""
     values:
-      fedora_release: "33"
+      fedora_release: "34"
```

Comment:
Maybe remove this as it's not being used i believe??

Reply:
This cannot be removed as some pages use this (for example copr-cli.md).